apps/nuxt3-ssr/pages/[schema]/ssr-catalogue/[catalogue]/networks/index.vue

```diff
@@ -142,7 +142,11 @@ let activeName = ref("detailed");
                 class="flex xl:hidden"
                 v-model:activeName="activeName"
               >
-                <FilterSidebar title="Filters" :filters="filters" />
+                <FilterSidebar
+                  title="Filters"
+                  :filters="filters"
+                  :mobileDisplay="true"
```

Comment:
the need to set this suggests that SearchResultsViewTabMobile ( 'mobile' ) should have its own template instead of passing through a slot and setting a mobile prop

Reply:
I agree but I copied this from 'cohorts' so I suppose merge for now